### DISCUSSION TITLE
Programa "Análise Portfa"

### DISCUSSION THREAD

**Discussion by tarciozemel** [frontendbr/forum#1439] [Dicas]
Inauguramos recentemente um novo programa em nosso canal no YouTube, o "Análise Portfa", em que analisamos e damos dicas de portfolios enviados pelos colegas.

Par quem quiser conhecer o formato, o primeiro episódio está aqui: https://www.youtube.com/watch?v=7UahNPEpX54

Vamos analisar portfolios de diversas áreas de desenvolvimento web, inclusive portfas de programadores. Por isso achei pertinente divulgar aqui, é um assunto correlato bem importante; infelizmente, devs brasucas não estão lá com os portfolios tão bons...

Enfim, para quem gostou e quiser dar aquela força, agradecemos de antemão! <3

Quem se interessou e quer ver o portfolio analisado no programa, basta preencher aqui: https://bit.ly/analise-portfa-form

**Comment by SrVieira** [comment]
Ótima iniciativa brother, logo logo mando o meu (assim que eu terminar) para avaliação em hehe

**Comment by tarciozemel** [comment]
Opa, ficamos no aguardo! 👍

**Comment by phenriqueleao** [comment]
Excelente iniciativa, ganhou um inscrito.
